src/fidesops/service/drp/drp_fidesops_mapper.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict
+
+from fidesops.models.privacy_request import PrivacyRequestStatus
+from fidesops.schemas.drp_privacy_request import DrpIdentity
+from fidesops.schemas.privacy_request import PrivacyRequestDRPStatus
+from fidesops.schemas.redis_cache import PrivacyRequestIdentity
+
+
+class DrpFidesopsMapper:
+    """
+    Map DRP objects/enums to Fidesops
+    """
+
+    @staticmethod
+    def map_identity(drp_identity: DrpIdentity) -> PrivacyRequestIdentity:
+        """
+        Currently, both email and phone_number identity props map 1:1 to the corresponding
+        Fidesops identity props in PrivacyRequestIdentity. This may not always be the case.
+        This class also allows us to implement custom logic to handle "verified" id props.
+        """
+        identity_kwargs = {
+            "email": drp_identity.email,
+            "phone_number": drp_identity.phone_number,
+        }
+        return PrivacyRequestIdentity(**identity_kwargs)
```

Comment:
Do we want to discard any extra attributes on `DrpIdentity` silently?

Reply:
I'll surface a warning log message using a validator or a check within this mapper